omnibus_overrides.rb

```diff
@@ -10,3 +10,6 @@
 override :rubygems, version: "2.6.8"
 # This SHA is the last commit before the 6.0 release
 override :'berkshelf-no-depselector', version: '6016ca10b2f46508b1b107264228668776f505d9'
+
+override :'postgresql-bin-versioned', version: "9.2.21" # needed for upgrading
```

Comment:
[minor] I wonder if we want to put the major version in the package name.  That is, this package would be called `postgresql92-bin`.